django_project/frontend/src/pages/Admin/RelatedTable/List/Attributes.jsx

```diff
@@ -43,7 +43,7 @@ export const relatedTableColumns = [
     field: 'created_at', headerName: 'Created at', flex: 0.5,
     renderCell: (params) => {
       return formatDateTime(new Date(params.value))
-    }, type: 'date'
+    }, type: 'date', cellClassName: 'RelatedTable-CreatedAt'
```

Comment:
Instead adding this class, please check the modified at, as it is already elipsed